bcp47/library/Data/BCP47/Internal/Arbitrary.hs

```diff
@@ -44,7 +44,8 @@ alphaNumChars :: String
 alphaNumChars = alphaChars ++ numChars
 
 alphaChars :: String
-alphaChars = ['a' .. 'z'] ++ ['A' .. 'Z']
+alphaChars = filter (`notElem` "xX") allAlphaChars
```

Comment:
It's surprising IMO to find a generic extension function named `(Internal.)Arbitrary.alphaChars` happens to avoid `xX` because it happens to be problematic in our specific use case.

Is it super cumbersome to make the caller do:

```hs
_ <- elements alphaChars `suchThat` (`notElem` "xX")
```

At the use-site?

That would also have the benefit of indicating that it's important closer to where it's important.

Reply:
Or `elements $ filter (`notElem` "xX") alphaChars` to avoid the nondeterminism that comes from `suchThat`

Reply:
The `complete` combinator actually fixed this, I don't need it at all